web/themes/custom/novel/templates/components/nav-teaser.html.twig

```diff
@@ -0,0 +1,19 @@
+<article {{ attributes.addClass('nav-teaser') }}>
+  <a href="{{ url }}">
+    <h1 class="nav-teaser__title">{{ title }}</h1>
+
+    {% if subtitle %}
+      <p class="nav-teaser__subtitle">{{ subtitle }}</p>
+    {% endif %}
+
+    <svg
```

Comment:
In other components we are using source.
We shouln't use svg's like this directly. 

An exmaple is eventinstance--list-teaser 

`  {{ source(directory ~ '/assets/dpl-design-system/icons/arrow-ui/icon-arrow-ui-small-right.svg') }}`

Reply:
👍